documentation/install_from_packages.markdown

```diff
@@ -7,103 +7,58 @@ canonical: "/puppetserver/latest/install_from_packages.html"
 
-## Platforms with Packages
-
-Puppet provides official packages that install Puppet Server 5.1 and all of its prerequisites on x86_64 architectures for the following platforms, as part of [Puppet Platform][repodocs].
-
-### Red Hat Enterprise Linux
-
--   Enterprise Linux 6
--   Enterprise Linux 7
-
-### Debian
-
--   Debian 8 (Jessie)
-
-Java 8 runtime packages do not exist in the standard repositories for Debian 8 (Jessie).  To install Puppet Server on Jessie, [configure the `jessie-backports` repository](https://backports.debian.org/Instructions/), which includes openjdk-8:
-
-```
-echo "deb http://ftp.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
-apt-get update
-apt-get -t jessie-backports install "openjdk-8-jdk-headless"
-apt-get install puppetserver
-```
-
-If you upgraded on Debian from older versions of Puppet Server, or from Java 7 to Java 8, you must also configure your server to use Java 8 by default for Puppet Server 5.x:
-
-```
-update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
-```
-
-### Ubuntu
-
--   Ubuntu 18.04 (Bionic)
--   Ubuntu 16.04 (Xenial)
-
-On Ubuntu 18.04, enable the [universe repository](https://help.ubuntu.com/community/Repositories/Ubuntu), which contains packages necessary for Puppet Server.
-
-### SUSE Linux Enterprise Server
-
--   SLES 12 SP1
-
-## Quick Start
+## Install Puppet Server from packages
 
 1.  [Enable the Puppet package repositories][repodocs], if you haven't already done so.
-2.  Stop the existing Puppet master service. The method for doing this varies depending on how your system is set up.
-
-    If you're running a WEBrick Puppet master, use: `service puppetmaster stop`.
-
-    If you're running Puppet under Apache, you'll instead need to disable the puppetmaster vhost and restart the Apache service. The exact method for this depends on what your Puppet master vhost file is called and how you enabled it. For full documentation, see the [Passenger guide][passengerguide].
-
-    -   On a Debian system, the command might be something like `sudo a2dissite puppetmaster`.
-    -   On RHEL or CentOS systems, the command might be something like `sudo mv /etc/httpd/conf.d/puppetmaster.conf ~/`. Alternatively, you can delete the file instead of moving it.
-
-    After you've disabled the vhost, restart Apache, which is a service called either `httpd` or `apache2`, depending on your OS.
 
-    Alternatively, if you don't need to keep the Apache service running, you can stop Apache with `service httpd stop` or `service apache2 stop`.
-
-3.  Install the Puppet Server package by running:
+2.  Install the Puppet Server package by running:
 
         yum install puppetserver
 
-    Or
+    or
 
         apt-get install puppetserver
 
     Note that there is no `-` in the package name.
 
-4.  Start the Puppet Server service:
+3.  Start the Puppet Server service:
+
+`systemctl start puppetserver` or `service puppetserver start`
```

Comment:
In the GH render I looked at this was also rendered as a regular sentence with a line break and monospaced sections. While the above `yum install ...` or `apt-get install ...` are both on their own lines broken out. I think having those two pairs rendered/treated differently is confusing. I don't care which (inline or broken out) but they're both CLI commands that vary across OSes and I think should be treated the same.